config/action.d/iptables.conf

```diff
@@ -98,11 +106,12 @@ _chain_rule = -p $proto <rule-jump>
 
 [Init]
 
-# Option:  chain
-# Notes    specifies the iptables chain to which the Fail2Ban rules should be
-#          added
+# Option:  chains
+# Notes    specifies the iptables chains to which the Fail2Ban rules should be
+#          added. May be a sigle chain (eg. INPUT) or a comma separated list
+#          (eg. INPUT, FORWARD)
 # Values:  STRING  Default: INPUT
-chain = INPUT
+chains = INPUT
```

Comment:
> the patch is simple and add the generic feature of applying ban rules to a list of chains

Well, it is... But this rename makes the action (and its derivates) backwards incompatible - so better would be to rename it back (as any others parameters like `port`, `protocol` etc that are singular).

Reply:
Your are absolutely right.